lib/wrappers/transform-node.js

```diff
@@ -13,6 +14,44 @@ module.exports = class TransformNodeWrapper extends NodeWrapper {
       cachePath: this.cachePath,
     });
     this.callbackObject = this.nodeInfo.getCallbackObject();
+    this.callbackObject.revised = () => {
+      this.revised();
+    };
+
+    // This weakmap holds references from inputNode --> last known revision #
+    // If the any inputNode's ref does not match what is stored in here then we
+    // know a modification has happened so we call the build method
+    this.inputRevisions = new WeakMap();
+  }
+
+  // Memoization is currently optin via either BROCCOLI_VOLATILE = true or
+  // by a plugin specifing that they are volatile
+  memoizationEnabled(volatile) {
```

Comment:
This doesn't seem like a prototype method, can we make this a help function at the module root instead?